examples/statistics/errorbars_and_boxes.py

```diff
@@ -0,0 +1,53 @@
+'''
+Example to create boxes from error using PatchCollection
+'''
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.collections import PatchCollection
+from matplotlib.patches import Rectangle
+
+# Number of data points
+n = 5
+
+# Dummy data
+x = np.arange(0, n, 1)
+y = np.random.rand(n)*5.
+
+# Dummy errors (above and below)
+xerr = np.random.rand(2, n)
+yerr = np.random.rand(2, n)
+
+# Create figure and axes
+fig, ax = plt.subplots(1)
+
+# Plot data points
+ax.errorbar(x, y, xerr=xerr, yerr=yerr, fmt='None', ecolor='k')
+
+
+def makeErrorBoxes(xdata, ydata, xerror, yerror, fc='r', ec='None', alpha=0.5):
+    '''
+    Function to create error boxes
+    '''
```

Comment:
I forget, did we want examples to have functions like this when it is just being called once? I thought it was only desirable to do for the examples to de-duplicate code?